lib/tests/testscripts/scripts/spec/testscript-example-setup-teardown.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestScript xmlns="http://hl7.org/fhir">
+    <id value="testscript-example-setup-teardown"/>
+
+    <url value="http://example.com"/>
+    <name value="Individual Patient Multisystem TestScript for Evaluating Measures--XML"/>
+    <status value="draft"/>
+    <date value="2019-02-11"/>
+    <publisher value="MITRE"/>
+    <contact>
+        <name value="Matthew Gramigna"/>
+        <telecom>
+            <system value="email"/>
+            <value value="mgramigna@mitre.org"/>
+            <use value="work"/>
+        </telecom>
+    </contact>
+    <description value="Tests using XML format to execute the $evauate-measure operation. The destination server must support the $evaluate-measure operation on the Measure resource."/>
+    <copyright value="N/A"/>
+
+    <fixture id="fixture-create-patient">
+        <resource>
+            <reference value="./_reference/resources/patient-example.json"/>
+        </resource>
+    </fixture>
+
+    <variable>
+        <name value="createPatientId"/>
+        <defaultValue value="example" />
+        <sourceId value="fixture-create-patient"/>
+    </variable>
+
+    <setup>
+        <action>
+            <operation>
+                <type>
+                    <system value="http://hl7.org/fhir/testscript-operation-codes"/>
+                    <code value="update"/>
+                </type>
+                <resource value="Patient"/>
+                <description value="Create a patient on the first server"/>
+                <accept value="json"/>
+                <contentType value="json"/>
+                <params value="/${createPatientId}"/>
```

Comment:
Is the forward slash in the `value` element here supposed to be there?

Reply:
See above